src/comm/AcceptLimiter.h

```diff
@@ -11,13 +11,13 @@
 
 #include "comm/TcpAcceptor.h"
 
-#include <vector>
+#include <deque>
 
 namespace Comm
 {
 
 /**
- * FIFO Queue holding listener socket handlers which have been activated
+ * Deque holding listener socket handlers which have been activated
```

Comment:
Let's restore `FIFO` to document the order. You do not have to modify this comment line at all AFAICT: The words "FIFO queue" are a high-level description of how deferred listeners are stored/ordered. The specific C++ type used to implement that storage does not matter here.